### ISSUE TITLE
headers can be lowercase

### ISSUE THREAD

**Issue by walterdeboer** [j-zeng/angular2-http-file-upload#11]
In chrome 60 XHR headers will be returned in lowercase, see https://stackoverflow.com/questions/45415480/xhr-getallresponseheaders-does-not-return-headers-as-expected-in-chrome-60

These are not recognised by the Uploader since it expects Content-Type instead of content-type

**Comment by walterdeboer** [comment]
See https://github.com/j-zeng/angular2-http-file-upload/pull/10, though I would just add a toLowerCase() to the fields in parseHeaders() and user lowercase headers everywhere...

**Comment by deanproxy** [comment]
Yeah, this may be a better way. I went for the least intrusive way to get it to work for my condition.  I think your suggestion may be a bit more robust.
